tensilelite/Tensile/TensileCreateLibrary.py

```diff
@@ -927,12 +732,7 @@ def validLogicFile(p: Path):
   if globalParameters["SeparateArchitectures"]:
     theMasterLibrary = list(masterLibraries.values())[0]
 
-  def checkFileExistence(files):
-    for filePath in files:
-      if not os.path.exists(filePath):
-        printExit(f"File {filePath} is missing.")
-
-  checkFileExistence(itertools.chain(libMetadataPaths, sourceLibPaths, asmLibPaths))
+  print1("# Check if generated files exists.")
```

Comment:
What purpose is this serving now that we're not calling `checkFileExistence`?

Reply:
It was removed in a previous commit and somehow added back in... probably from all of the merge conflict resolution...